patches/tModLoader/Terraria/DataStructures/GameModeData.TML.cs

```diff
@@ -0,0 +1,72 @@
+using Terraria.GameContent.Creative;
+
+namespace Terraria.DataStructures
+{
+	public partial class GameModeData
+	{
+		public float GetPureEnemyDamageModifier() {
+			float enemyDamageModifier = EnemyDamageMultiplier;
+
+			if(IsJourneyMode) {
+				CreativePowers.DifficultySliderPower difficultySliderPower =
+					CreativePowerManager.Instance.GetPower<CreativePowers.DifficultySliderPower>();
+				enemyDamageModifier *= difficultySliderPower.StrengthMultiplierToGiveNPCs;
+			}
+			
+			return enemyDamageModifier;
+		}
+
+		public float GetEnemyDamageModifier() {
```

Comment:
I think these should probably be properties, and they should have the original name with `Effective` in front of them.